### PR TITLE
Update NanoPlot to 1.41.6

### PR DESCRIPTION
**Pull request by kprus**
Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

Updated NanoPlot to v1.41.6

### CODE REVIEW

File: nanoplot/1.41.6/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# NanoPlot container
+
+Main tool : [NanoPlot](https://github.com/wdecoster/NanoPlot)
+
+Additional tools:
+
+- nanomath 1.3.0
+- nanoget 1.19.3
```
Comment (erinyoung) [inline nanoplot/1.41.6/README.md:8]: Note to self. I looked these up and these are the current versions of nanomath and nanoget.

**Comment by erinyoung** [comment]
This looks great! I don't have any changes to recommend. 

Did you want to add your handle to the bottom of [docker-builds/README.md](https://github.com/kprus/docker-builds/edit/master/README.md)?

If not, I'll merge your PR and push this image to Dockerhub and quay.

**Comment by erinyoung** [comment]
This PR will close https://github.com/StaPH-B/docker-builds/issues/751

**Comment by kprus** [comment]
I added my handle to the README so should be good to go now. Thanks!

**Review by erinyoung — APPROVED** [review]
Looks great!

I am going to 
- merge this PR
- deploy this image for nanoplot with the tag 1.41.6 to both quay and dockerhub
- overwrite the 'latest' tag

**Comment by erinyoung** [comment]
You can watch the status of the deployment here : https://github.com/StaPH-B/docker-builds/actions/runs/6644683675

Dockerhub image : https://hub.docker.com/r/staphb/nanoplot
quay image : https://quay.io/repository/staphb/nanoplot

**Comment by erinyoung** [comment]
Thank you for your contribution!!! (And for finishing the docker workshop homework)

**Pull request merged**